charts/kyverno/templates/reports-server/_helpers.tpl

```diff
@@ -0,0 +1,90 @@
+{{/*
+Reports Server helpers
+*/}}
+
+{{/*
+Check if reports-server is enabled
+*/}}
+{{- define "kyverno.reportsServer.enabled" -}}
+{{- if .Values.reportsServer.enabled -}}
+true
+{{- else -}}
+false
+{{- end -}}
+{{- end -}}
+
+{{/*
+Reports Server readiness init container
+*/}}
+{{- define "kyverno.reportsServer.initContainer" -}}
+{{- if and .Values.reportsServer.enabled .Values.reportsServer.waitForReady }}
+- name: wait-for-reports-server
+  image: {{ include "kyverno.image" (dict "globalRegistry" .Values.global.image.registry "image" .Values.test.image "defaultTag" .Values.test.image.tag) | quote }}
+  imagePullPolicy: {{ .Values.test.image.pullPolicy | default "IfNotPresent" }}
+  command:
+    - /bin/sh
+    - -c
+    - |
+      echo "Waiting for reports-server to be ready..."
+      timeout={{ .Values.reportsServer.readinessTimeout | default 300 }}
+      elapsed=0
+      
+      # Check if kubectl is available
+      if ! command -v kubectl >/dev/null 2>&1; then
+        echo "kubectl not available in this image, using basic readiness check"
```

Comment:
Is `kubectl` expected to be part of reports-server image? Or can we use our own kubectl image?